core/services/blockhashstore/delegate.go

```diff
@@ -199,6 +201,7 @@ type service struct {
 	utils.StartStopOnce
 	feeder     *Feeder
 	done       chan struct{}
```

Comment:
Is this channel used anywhere else? If not it can be removed.

Reply:
Removed `done chan struct{}`